src/Init/Data/Int/Bitwise/Lemmas.lean

```diff
@@ -0,0 +1,48 @@
+/-
+Copyright (c) 2023 Siddharth Bhat. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Siddharth Bhat, Jeremy Avigad
+-- https://github.com/leanprover-community/mathlib4/blob/12b9e3064e35636d35f0c349bf2be29fb9b75bf2/Mathlib/Data/Int/Bitwise.lean#L490-L490
```

Comment:
How precisely should I tag the fact that these were taken from mathlib and adapted here?

Reply:
I think as long as you copied across the authors field (✅ ), you're fine. It wouldn't hurt to have a module doc:

```
/-!
# Lemmas about bitwise operations on `Int`

Some of these lemmas have been adapted from existing material in Mathlib.
-/